src/components/MeasurementsToolList.vue

```diff
@@ -135,17 +182,42 @@ function toggleGlobalHidden() {
       <v-btn
         icon
         variant="text"
-        :disabled="!selectedSome"
+        :disabled="selectionState === MultipleSelectionState.None"
         @click.stop="removeAll"
       >
         <v-icon>mdi-delete</v-icon>
-        <v-tooltip :disabled="!selectedSome" location="top" activator="parent">
+        <v-tooltip
+          :disabled="selectionState === MultipleSelectionState.None"
+          location="top"
+          activator="parent"
+        >
           Delete selected
         </v-tooltip>
       </v-btn>
     </v-col>
   </v-row>
 
+  <v-list-item v-if="selectedIDsFromOtherImages.length">
+    <v-container>
```

Comment:
No feature request yet. There are two options I considered: clearing the selection when the primary selection changes, or keeping it. I initially went with keeping it due to the selection possibly spanning multiple images (e.g. if we ever show multiple images at once), but since we haven't moved towards that implementation I'm now leaning towards clearing it.